src/Footer/Footer.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import classes from "./Footer.module.css";
+import PropTypes from "prop-types";
+
+const Footer = ({
+  onListSelect,
+  onAddSelect,
+  isAddSelected,
+  isListSelected
+}) => {
+  return (
+    <footer className={classes.footer}>
+      <NavLink to="/" exact>
+        <button
+          onClick={onListSelect}
+          className={isListSelected ? classes.active : classes.nonactive}
```

Comment:
Nice use of a ternary here! React goes so well with these sorts of short expressions.

Reply:
Thanks! Love JSX.

Reply:
I think I actually would like an example, please! I'm turning in PR requests in my internship and I'm getting no guidance or feedback, but they're merging, so I have no idea what I need to communicate. I appreciate your help!